egs/wham/ConvTasNet/run.sh

```diff
@@ -1,33 +1,17 @@
 #!/bin/bash
 
 # Exit on error
-set -e
-set -o pipefail
-
-# Main storage directory. You'll need disk space to dump the WHAM mixtures and the wsj0 wav
-# files if you start from sphere files.
-storage_dir=
-
-# If you start from the sphere files, specify the path to the directory and start from stage 0
-sphere_dir=  # Directory containing sphere files
-# If you already have wsj0 wav files, specify the path to the directory here and start from stage 1
-wsj0_wav_dir=
-# If you already have the WHAM mixtures, specify the path to the directory here and start from stage 2
-wham_wav_dir=
-# After running the recipe a first time, you can run it from stage 3 directly to train new models.
-
-# Path to the python you'll use for the experiment. Defaults to the current python
-# You can run ./utils/prepare_python_env.sh to create a suitable python environment, paste the output here.
-python_path=python
-
-# Example usage
-# ./run.sh --stage 3 --tag my_tag --task sep_noisy --id 0,1
+set -eo pipefail
 
 # General
 stage=0  # Controls from which stage to start
-tag=""  # Controls the directory name associated to the experiment
+tag=1a_wham  # Controls the directory name associated to the experiment
+
+# GPU setting.
 # You can ask for several GPUs using id (passed to CUDA_VISIBLE_DEVICES)
-id=$CUDA_VISIBLE_DEVICES
+#   For single gpu, "0" or "1" would work
+#   For multi-gpu, something like "0,1"
+id=0
```

Comment:
Why? 
By default, this uses all available GPUs, why is it wrong?

Reply:
Tried on grid5k and seems not the case for me. But it works on the cluster from my school. Here's result on grid5k:

```
$ nvidia-smi
Fri Feb  5 19:13:32 2021
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 450.80.02    Driver Version: 450.80.02    CUDA Version: 11.0     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  GeForce GTX TIT...  On   | 00000000:82:00.0 Off |                  N/A |
| 26%   23C    P8    13W / 250W |      1MiB /  6083MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
$ echo $CUDA_VISIBLE_DEVICES

$ echo $CUDA_VISIBLE_DEVICES
```

Reply:
The $CUDA_VISIBLE_DEVICES is set on most clusters I worked on. 
I don't think it's worth changing to a fixed id, which most of the time may be wrong.

Reply:
hmm....ok I will change it back.